src/xcode/ENA/ENA/Source/Scenes/Home/HomeViewController.swift

```diff
@@ -100,6 +103,11 @@ final class HomeViewController: UIViewController, RequiresAppDependencies {
 		homeInteractor.requestRisk(userInitiated: false)
 		updateBackgroundColor()
 	}
+	
+	override func viewWillDisappear(_ animated: Bool) {
+		let notificationCenter = NotificationCenter.default
+        notificationCenter.removeObserver(self, name: UIApplication.didBecomeActiveNotification, object: nil)
```

Comment:
Maybe this is intended, but I think adding the observer in viewDidLoad and removing it in viewWillDisappear could lead to a ViewController which does not react to the notification.

If you remove it in viewWillDisappear you should add it in viewWillAppear.
Or keep it in viewDidLoad and remove it in deinit.

Calls can be:
viewDidLoad -> viewWillAppear -> viewWillDisappear -> viewWillAppear and so on.

Reply:
Hi Artur, thanks for that note. To be consistent, I will move the code from `viewWillDisappera` to `deinit`.

Reply:
some remark:
If you're using `selector` to do the callback, you don't need to do the unregister. From Apple:

**If your app targets iOS 9.0 and later or macOS 10.11 and later, you do not need to unregister an observer that you created with this function**

Reply:
Thanks @haosap for that. I removed `deinit` now completely